framework/py/flwr/common/inflatable_grpc_utils.py

```diff
@@ -94,10 +95,14 @@ def pull_object_from_servicer(
 ) -> InflatableObject:
     """Recursively inflate an object by pulling it from the servicer."""
     # Pull object
-    object_proto: PullObjectResponse = stub.PullObject(
-        PullObjectRequest(node=node, run_id=run_id, object_id=object_id)
-    )
-    object_content = object_proto.object_content
+    object_available = False
+    while not (object_available):
+        object_proto: PullObjectResponse = stub.PullObject(
+            PullObjectRequest(node=node, run_id=run_id, object_id=object_id)
+        )
+        object_available = object_proto.object_available
+        object_content = object_proto.object_content
+        sleep(0.1)
```

Comment:
We might need a way to stop this. The object requested might never become available.